MAUI/ImageEditor/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Overview of .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control, its basic features and functionalities of imageeditor.

```

Comment:
Description: Learn about the introduction of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control, its basic features, and the functionalities of imageeditor.

Reply:
content updated